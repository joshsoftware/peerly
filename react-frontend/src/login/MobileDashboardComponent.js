import React from "react";
import styled from "styled-components";

import PeerlyTextAndLogo from "shared-components/peerly-logo/PeerlyTextAndLogo";
import InformativeTextComponent from "login/InformativeTextComponent";
import LoginImageComponent from "login/LoginImageComponent";

const Wrapper = styled.div`
  height: 100vh;
  background-color: var(--atomic);
`;

const MobileDashboardComponent = () => (
  <Wrapper className="d-md-none flex-column d-flex justify-content-around">
    <PeerlyTextAndLogo theme="dark" />
    <LoginImageComponent />
    <InformativeTextComponent
      className="mx-5"
      theme="dark"
      informativeText="Lets Create the Office Positive"
      encouragementThought="Encouragement Driven"
    />
  </Wrapper>
);

export default React.memo(MobileDashboardComponent);
