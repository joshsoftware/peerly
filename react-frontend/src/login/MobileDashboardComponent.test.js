import React from "react";
import { render } from "@testing-library/react";
import MobileDashboardComponent from "login/MobileDashboardComponent";

describe("mobile dashboard component test", () => {
  test("should render  dashboard component", () => {
    const { asFragment } = render(<MobileDashboardComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
