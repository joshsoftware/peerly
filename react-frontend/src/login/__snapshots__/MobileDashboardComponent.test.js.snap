// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mobile dashboard component test should render  dashboard component 1`] = `
<DocumentFragment>
  .c2 {
  -webkit-transform: rotate(45deg);
  -ms-transform: rotate(45deg);
  transform: rotate(45deg);
  font-size: 30px;
  color: var(--white);
}

.c1 {
  text-align: center;
  margin-top: 15px;
  width: 50px;
  height: 50px;
  border: 3px solid var(--white);
  background-color: var(--atomic);
  -webkit-transform: rotate(45deg);
  -ms-transform: rotate(45deg);
  transform: rotate(45deg);
}

.c3 {
  color: var(--white);
  background-color: var(--atomic);
  font-size: 30px;
}

.c5 {
  width: 10%;
  background: var(--white);
}

.c4 {
  color: var(--white);
  background-color: var(--atomic);
}

.c0 {
  height: 100vh;
  background-color: var(--atomic);
}

<div
    class="c0 d-md-none flex-column d-flex justify-content-around"
  >
    <div
      class="text-center"
    >
      <div
        class="c1 mx-auto"
        data-testid="Logo"
      >
        <div
          class="c2"
          data-testid="plusSignText"
          font-size="30px"
        >
          +
        </div>
      </div>
      <div
        class="c3"
        data-testid="PeerlyTextComponents"
        font-size="30px"
      >
        Peerly
      </div>
    </div>
    <img
      alt="login image"
      class="img-fluid"
      src="/assets/images//cat-mobile-img.png"
    />
    <div
      class="c4 text-center mx-5"
      data-testid="info"
    >
      <h3>
        Lets Create the Office Positive
      </h3>
      <hr
        class="c5"
        data-testid="hrLine"
      />
      <h6>
        Encouragement Driven
      </h6>
    </div>
  </div>
</DocumentFragment>
`;
